src/app/components/crypto-assets/stacks/stx20-token-asset-list/stx20-token-asset-list.tsx

```diff
@@ -0,0 +1,12 @@
+import type { Stx20Token } from '@app/api/stacks/types/stx20-types';
+
+import { Stx20TokenAssetItemLayout } from './stx20-token-asset-item.layout';
+
+interface Stx20TokenAssetListProps {
+  stx20Tokens: Stx20Token[];
```

Comment:
nit, for me it's clear enough from component name this is for stx20, so just "tokens" is good as a prop name?